search-service/src/main/kotlin/com/egm/stellio/search/entity/model/EntitiesQuery.kt

```diff
@@ -16,6 +16,7 @@ sealed class EntitiesQuery(
     open val datasetId: Set<String>,
     open val geoQuery: GeoQuery?,
     open val linkedEntityQuery: LinkedEntityQuery?,
+    open val local: Boolean?,
```

Comment:
can't we have a default to `false` (when not provided, it is false) and avoid having a nullable value?